src/components/SearchAndFilter/SearchAndFilter.tsx

```diff
@@ -6,28 +6,28 @@ import { filterProps, filterUpdate, stringKeyOptions } from './definitions'
 
 import './SearchAndFilter.css'
 
-const reduceSelectedFilters = (state: object, filterUpdate:filterUpdate) => {
+const changeSelectedFilters = (state: object, filterUpdate:filterUpdate) => {
```

Comment:
This name is still a little misleading, because the function is also used to create the selected filters. It's a problem with SRP, but I'd prefer to keep the reducer tight and lean. I don't imagine `createSelectedFilters` would look all that different.